lib/components/pagination-nav.vue

```diff
@@ -90,7 +90,7 @@
             </li>
             <li v-else class="page-item" role="none presentation">
                 <b-link class="page-link"
-                        v-bind="linkProps(currentPage - 1)"
+                        v-bind="linkProps(numberOfPages)"
```

Comment:
Everything else works fine, linkGen func awsome, but it need detailed docs. That part about using arrays of links and assigning length-1 to total pages, and pagenum-1, especially.